src/style.css

```diff
@@ -1,3 +1,69 @@
-body {
+:root {
+  --current-text: #000;
+  --brend-color: #00acdc;
+}
+
+.body {
+  margin: 0;
+  font-family: "Roboto", "Arial", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
   margin: 0;
+  padding: 0;
+}
+
+.nav__list__link,
+a[href^="#laptops"] {
```

Comment:
why do you need this tag here

Reply:
becuse i cant add class to tag "a" because "29:48 error Only 2 attributes per line are permitted attr-new-line "